src/main/java/org/recordrobotics/charger/RobotMap.java

```diff
@@ -4,6 +4,17 @@
  * Hardware ports for computer and robot
  */
 public class RobotMap {
+	/**
+	 * Arm Ports (Phoenix) - will change once decided
+	 * Ports will also be in SIM/Neo - talk to Jake for more info
+	 */
+	public class Arm {
+		/** the motor that will be (0,0) in the calculation */
+		public static final int ORIGIN_MOTOR_PORT = 1;
```

Comment:
I doubt these should be 1 and 2.
Please create a port scheme which will make sense, as ports can be changed with the software.

Reply:
these are placeholders. when we get the actual port numbers and figure it out, we'll change it accordingly.